shared/api-ref-ex.asciidoc

```diff
@@ -46,13 +46,12 @@ Guidelines for parameter documentation
 ***************************************
 * Use a definition list.
 * End each definition with a period.
-* Include the data type.
-* Include whether the parameter is Optional or Required.
+* Include the data type and whether the parameter is Optional or Required.
```

Comment:
Should we reverse the order here since the current template includes optional/required THEN data type.

Reply:
Done, thanks!